src/chrome/content/rules/searx.fi.xml

```diff
@@ -0,0 +1,8 @@
+<ruleset name="searx.fi">
+	<target host="searx.fi" />
+	<target host="www.searx.fi" />
```

Comment:
I get a `401` on both targets.

Reply:
@J0WI It was working before without authentication, I asked its operator and he said that he put it behind a password: https://github.com/asciimoo/searx/issues/1072#issuecomment-347153737

Do you think I should remove it nevertheless?

Reply:
I wounder why the tests are passing since we are not able to test this host.
I would remove it, yes.